src/Universe/Resources/BuildCosts.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EtoA\Universe\Resources;
+
+use EtoA\Universe\Planet\Planet;
+
+class BuildCosts
```

Comment:
Wonder whether this shouldn't be merged with `PreciseResources` otherwise we'll end up with multiple very similar objects. The only difference I see here is that `BuildCosts` has a power property and `PreciseResources` people property. Power does not get stored or can be paid so this shouldn't be necessary here anyway.